pallets/liquidity-pools/src/tests.rs

```diff
@@ -545,6 +556,27 @@ mod add_tranche {
 				);
 			})
 		}
+
+		#[test]
+		fn with_no_hook_address() {
+			System::externalities().execute_with(|| {
+				Permissions::mock_has(|_, _, _| true);
+				Pools::mock_pool_exists(|_| true);
+				Pools::mock_tranche_exists(|_, _| true);
+				AssetRegistry::mock_metadata(|_| Some(util::default_metadata()));
```

Comment:
NIT (I think it comes from a merge where does not applied the config_mocks() call)
```suggestion
				config_mocks();
```

Reply:
In `config_mocks()`, we mock a hook address for the `AddTranche` message. This test requires this hook to not have been set. If we execute `config_mocks`, then we need to revert https://github.com/centrifuge/centrifuge-chain/pull/1909/files/5a63633955ba13282a2448dd2a15262e5173c9ba#diff-57577a9a8bc154a3e3b81af63e11d2b863c6716120411b1480117c8c44ee02d5R446-R452

Reply:
The idea is to overwrite the mock make it fail, so:
```rust
config_mocks();
Gateway::mock_get(|_| None);
```
make the tests work as expected.

The idea is not to copy-past all initialization for all these tests and not create an order in how the mocks are defined. I did that refactor, but I think this test was added later and was skipped by me in the merge.

Reply:
I changed it in a commit after for your work on this, so you indeed followed the status-quo!!